cmake/ExternalProjects.cmake

```diff
@@ -111,7 +111,7 @@ FetchContent_Declare(wavm_ext
 
 FetchContent_Declare(wamr_ext
     GIT_REPOSITORY "https://github.com/faasm/wasm-micro-runtime"
-    GIT_TAG "135672cab24d877db4ef6933c0ab150351384d51"
+    GIT_TAG "a31e5a4fa299c4f8384f40e157b0a928ad0bda1b"
```

Comment:
Unfortunately, in a PR where I am working on SGX (#686) I rebased the `faasm` branch to the latest WAMR commit, effectively removing the previous tag. It does not exist anymore.

As a future note to self, any other time I rebase the WAMR dependency I will do it in a temporary branch whilst the PR is under development.